backend/matchmaking/consumers.py

```
@@ -97,10 +97,51 @@ async def match_player(self):
 
             if player_in_queue:
                 if await player_in_queue.has_player_been_matched():
+                    assigned_game_id = player_in_queue.assigned_game_id
+                    assigned_color = player_in_queue.assigned_color
+
+                    white_player = player_in_queue if assigned_color and assigned_color.lower() == "white" else matched_user
```

Pylint (3.3.7)

[convention] Line too long (124/100)

(C0301)